## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 452734`
Commit: [`857beb6d531e914028559ee1452740058ebc2b9c`](https://github.com/NixOS/nixpkgs/commit/857beb6d531e914028559ee1452740058ebc2b9c) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/857beb6d531e914028559ee1452740058ebc2b9c..pull/452734/head))
Merge: [`bcf632f76ec11f7230bb6d9fc34d7525c9419569`](https://github.com/NixOS/nixpkgs/commit/bcf632f76ec11f7230bb6d9fc34d7525c9419569)

Logs: https://github.com/Sigmanificient/nixpkgs-review-gha/actions/runs/18600620535


---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>cups-zj-58</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>cups-zj-58</summary>
<pre>unpacking source archive /nix/store/p2jcywxvj3b0pknrknjxsv8hriqwr399-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/share/doc/rastertozj -DCMAKE_INSTALL_INFODIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/include -DCMAKE_INSTALL_SBINDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/186qghi5ddn8hh6c7ygkjv9zfm60y266-cups-zj-58-2019-04-28
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake &lt; 3.5 has been removed from CMake.

  Update the VERSION argument &lt;min&gt; value.  Or, use the &lt;min&gt;...&lt;max&gt; syntax
  to tell CMake that the project requires at least &lt;min&gt; but has been updated
  to work with policies introduced by &lt;max&gt; or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>cups-zj-58</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>cups-zj-58</summary>
<pre>unpacking source archive /nix/store/p2jcywxvj3b0pknrknjxsv8hriqwr399-source
source root is source
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/share/doc/rastertozj -DCMAKE_INSTALL_INFODIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/include -DCMAKE_INSTALL_SBINDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/frxwlync78r3l94863r2cbnz7ngy273j-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/frxwlync78r3l94863r2cbnz7ngy273j-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/frxwlync78r3l94863r2cbnz7ngy273j-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/miwyznppajc57wi2zdrn3s3jbc0mbfg6-cups-zj-58-2019-04-28
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake &lt; 3.5 has been removed from CMake.

  Update the VERSION argument &lt;min&gt; value.  Or, use the &lt;min&gt;...&lt;max&gt; syntax
  to tell CMake that the project requires at least &lt;min&gt; but has been updated
  to work with policies introduced by &lt;max&gt; or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>tests.pkgs-lib</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>tests.pkgs-lib</li>
  </ul>
</details>

--- a/pkgs/by-name/cu/cups-zj-58/package.nix
+++ b/pkgs/by-name/cu/cups-zj-58/package.nix
@@ -9,13 +9,13 @@
 
 stdenv.mkDerivation {
   pname = "cups-zj-58";
-  version = "2019-04-28";
+  version = "0-unstable-2019-04-28";
 
   src = fetchFromGitHub {
     owner = "klirichek";
     repo = "zj-58";
     rev = "64743565df4379098b68a197d074c86617a8fc0a";
-    sha256 = "sha256-4l9NRfp0hiPDC6dtFsq7jLf0Gn9tktGy6oZ4GHxSfbw=";
+    hash = "sha256-4l9NRfp0hiPDC6dtFsq7jLf0Gn9tktGy6oZ4GHxSfbw=";
   };
 
   nativeBuildInputs = [

--- a/pkgs/by-name/cu/cups-zj-58/package.nix
+++ b/pkgs/by-name/cu/cups-zj-58/package.nix
@@ -25,6 +25,11 @@ stdenv.mkDerivation {
 
   buildInputs = [ cups ];
 
+  patchPhase = ''
+    substituteInPlace CMakeLists.txt \
+    --replace-fail "cmake_minimum_required ( VERSION 3.0 )" "cmake_minimum_required ( VERSION 3.10 )"
+  '';
+
   installPhase = ''
     install -D ppd/zj80.ppd $out/share/cups/model/zjiang/zj80.ppd
     install -D ppd/zj58.ppd $out/share/cups/model/zjiang/zj58.ppd

